src/math.rs

```diff
@@ -554,3 +555,26 @@ where
         Box::new(Frustum::new(matrix))
     }
 }
+
+// Returns transform needed to go from local frame to ECEF with the specified origin where
+// the axes are ENU (east, north, up <in the direction normal to the oblate spheroid
+// used as Earth's ellipsoid, which does not generally pass through the center of the Earth>)
+// https://en.wikipedia.org/wiki/Geographic_coordinate_conversion#From_ECEF_to_ENU
+pub fn local_frame_from_lat_lng(lat: f64, lon: f64) -> Matrix4<f64> {
+    const PI_HALF: Deg<f64> = Deg(90.0);
+    let lat_lng_alt = WGS84::new(lat, lon, 0.0);
+    let origin = ECEF::from(lat_lng_alt);
+    let origin_vector = Vector3::new(origin.x(), origin.y(), origin.z());
+    println!("origin_vector {:?}", origin_vector);
+    let rotation_matrix = Matrix3::from_angle_z(-PI_HALF)
+        * Matrix3::from_angle_y(Deg(lat_lng_alt.latitude_degrees()) - PI_HALF)
+        * Matrix3::from_angle_z(Deg(-lat_lng_alt.longitude_degrees()));
```

Comment:
I still have to understand this rotation thoroughly.. usually rot z, rot y and rot x are applied. from ECEF to ENU this font explains that two rotations suffice to get from ecef to enu and backwards. https://gssc.esa.int/navipedia/index.php/Transformations_between_ECEF_and_ENU_coordinates